src/data_storage/import/data_staging_storage.ts

```diff
@@ -63,9 +58,20 @@ export default class DataStagingStorage extends PostgresStorage {
     }
 
     public async List(importID: string, offset:number, limit:number): Promise<Result<DataStagingT[]>>{
+        if(limit === -1) { // allow return all with a negative number
+            return super.rows<DataStagingT>(DataStagingStorage.listAllStatement(importID))
+        }
         return super.rows<DataStagingT>(DataStagingStorage.listStatement(importID, offset, limit))
     }
 
+    public async ListAndSort(importID: string, offset:number, limit:number, sortBy:string, sortDesc: boolean): Promise<Result<DataStagingT[]>>{
+        if(sortDesc) {
```

Comment:
Would it be easier/simpler to have one listSortStatement and pass sortDesc through to it? Either is fine with me though.

Reply:
I could do that, yeah